testsuite/pom.xml

```diff
@@ -28,12 +28,17 @@
         <version.wildfly10>10.0.0.Final</version.wildfly10>
         <version.wildfly10.core>2.0.10.Final</version.wildfly10.core>
         <version.wildfly10.arquillian>1.1.0.Alpha1</version.wildfly10.arquillian>
+        <!-- doesn't exist yet; update when there's a public release of WildFly 11 -->
+        <version.wildfly11>11.0.0.Alpha1</version.wildfly11>
+        <version.wildfly11.core>3.0.0.Alpha10</version.wildfly11.core>
+        <version.wildfly11.arquillian>2.0.0.Final</version.wildfly11.arquillian>
 
         <managementPort.as7>9999</managementPort.as7>
         <managementPort.wildfly>9990</managementPort.wildfly>
 
         <ignoredCategory.as7>org.wildfly.extras.creaper.test.WildFlyTests</ignoredCategory.as7>
```

Comment:
I think no, because SnapshotBackupTest is declared as Category {WildFlyTests, WildFly11Tests}